src/components/organelle-wrapper.tsx

```diff
@@ -83,17 +222,34 @@ class OrganelleWrapper extends React.Component<any, any> {
       this.addHormone();
     }
 
-    if (nextProps.addEnzyme) {
-      let cell: any = document.querySelector(`#${this.props.name} #cellshape_0_Layer0_0_FILL`);
-      cell.style.fill = 'rgb(177,122,50)';
+    // if (nextProps.addEnzyme) {
+    //   let cell: any = document.querySelector(`#${this.props.name} #cellshape_0_Layer0_0_FILL`);
+    //   cell.style.fill = 'rgb(177,122,50)';
+    // } else {
+    //   let cell: any = document.querySelector(`#${this.props.name} #cellshape_0_Layer0_0_FILL`);
+    //   cell.style.fill = 'rgb(241,212,151)';
+    // }
+  }
+
+  componentDidUpdate() {
```

Comment:
This was the big change I made to the `organelle-wrapper`. Previously, there were a lot of different places where the cell was made opaque and transparent, to the point where it was not easy to tell how the cell should look by looking at the React state. Now, the view of the cell is updated post-update and is tightly tied to the props and state of `organelle-wrapper`. As it stands, it might not be as efficient, since I think it could end up doing duplicate work making everything opaque, for example. What do you think about this approach?